docs/lua-modules/httpserver.md

```diff
@@ -5,6 +5,10 @@
 
 This Lua module provides a simple callback implementation of a [HTTP 1.1](https://www.w3.org/Protocols/rfc2616/rfc2616.html) server.
 
+!!!note
+	copy this module to nodemcu by yourself 
+
+
```

Comment:
This is true of all `lua_modules`, in general.

Reply:
at first time, I don't know that must be copy by myself. so add a note to tell others who don't know

Reply:
That's fair, but I think this disclaimer should be higher in the document hierarchy, perhaps where we introduce the `lua_modules`.

Reply:
agree